File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Request/GetEntriesByRaceRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ubb.BikeContest.Networking.Request
{
    [Serializable]
    public class GetEntriesByRaceRequest : IRequest
    {
        private readonly long raceId;

        public GetEntriesByRaceRequest(long raceId)
        {
            this.raceId = raceId;
        }

        public virtual long RaceId
        {
           get { return raceId; }
        }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab11/java/networking/src/main/java/com/ubb/protocol/request/GetEntriesByRaceRequest.java
package com.ubb.protocol.request;

public class GetEntriesByRaceRequest implements Request {
    private final Long raceID;

    public GetEntriesByRaceRequest(Long raceID) {
        this.raceID = raceID;
    }

    public Long getRaceID() {
        return raceID;
    }
}
